src/client/TargetsServerClient.js

```diff
@@ -14,39 +14,46 @@ class TargetClient {
     });
   }
 
+  static getMyTargets() {
+    return new Promise((resolve, reject) => {
+      instance.get('/targets').then((({ data })=> {
```

Comment:
Maybe you can rename this instance in a way that remains clear that it's an Axios instance?